presto-plugin-toolkit/src/main/java/com/facebook/presto/plugin/base/security/ReadOnlyAccessControl.java

```diff
@@ -144,6 +143,12 @@ public void checkCanCreateViewWithSelectFromView(ConnectorTransactionHandle tran
         // allow
     }
 
+    @Override
+    public void checkCanCreateViewWithSelectFromColumns(ConnectorTransactionHandle transactionHandle, Identity identity, SchemaTableName tableName, Set<String> columnNames)
```

Comment:
Shouldn't `checkCanCreateViewWithSelectFromColumns` go always right after `checkCanSelectFromColumns`?

Reply:
The other `checkCanCreateViewWithSelectFrom`s were next to each other and rather than their `checkCanSelectFrom` counterparts.  That seemed as good an organizing principle.